lib/bybe_utils.rb

```diff
@@ -777,16 +777,6 @@ def replace_with_redirect(m_id_to_delete, m_id_to_redirect_to)
     h.save
   end
 
-  def make_heading_ids_unique(html)
```

Comment:
This method duplicated method in ApplicationController and was used outside of controller context. Now it is in service so we can safely remove it.